skills/csharp/emailskill/Dialogs/EmailSkillDialogBase.cs

```diff
@@ -1270,13 +1267,62 @@ protected async Task ClearAllState(WaterfallStepContext sc)
         protected async Task DigestFocusEmailAsync(WaterfallStepContext sc)
         {
             var state = await EmailStateAccessor.GetAsync(sc.Context);
+            sc.Context.Activity.Properties.TryGetValue("OriginText", out var content);
```

Comment:
what is OriginText?

Reply:
Seems not be used now.. Removed all OriginText